src/element/types.ts

```diff
@@ -21,7 +21,7 @@ type _ExcalidrawElementBase = Readonly<{
   strokeWidth: number;
   strokeStyle: StrokeStyle;
   strokeSharpness: StrokeSharpness;
-  roughness: number;
+  roughness: 0 | 1 | 2;
```

Comment:
`const type Roughness = 0 | 1 | 2;` maybe?

Reply:
Not much reason. In fact I'm not sure why we're exporting the other types and not simply referencing them via `ExcalidrawElement[prop]` — that way the intent is clearer.

Reply:
Either this or that.. now we have a mix.. something for another PR though.